lib/controller.js

```diff
@@ -170,6 +177,134 @@ Tessel.get = function(opts) {
     }
   });
 };
+/*
+Because of sync problem with master trunk and parallel development on Tessel.list 
+it was necessary to go my own way...
```

Comment:
yes, if the [logAndFinish](https://github.com/tessel/t2-cli/blob/master/lib/controller.js#L162-L168)  got a menu and **returns the selected Tessel (or set the process.env.TESSEL)** the `controller.ssh.seek(opts)` could be replaced by `Tessel.get(opts)` in [controller.root](https://github.com/Student007/t2-cli/blob/t2-root/lib/controller.js#L285-L305) - I have to take a look at `Tessel.get` I am currently not familiar how it works like (comming soon ;) ).